CHANGELOG

```diff
@@ -1,6 +1,11 @@
 devel
 -----
 
+* APM-592: Added a retriable flag for queries in which, case the batch response
+  object isn't successfully received, by sending a POST request with 
+  `_api/cursor/<cursorId>/<batchId>`, the response object from the `batchd` that
+  had not been received before would be returned. The request is idempotent.
```

Comment:
Sorry, I am not able to parse that sentence. Can it be rephrased and simplified?